Traits/TestsTraits/PhpUnit/TestsAuthHelperTrait.php

```diff
@@ -187,8 +187,12 @@ private function setupTestingUserAccess($user, $access = null)
     private function setupTestingUserRoles($user, $access)
     {
         if (isset($access['roles']) && !empty($access['roles'])) {
-            if (!$user->hasRole($access['roles'])) {
-                $user->assignRole($access['roles']);
+
+            $roles = is_array($access['roles']) ? $access['roles'] :
+            explode('|', $access['roles']);
```

Comment:
please don't do this! - it took me 3min to figure out, that L191 and L192 is *one* single command, which is really bad formatted.. I thought you would `explode()` and never use the result... Same on the other function!

Reply:
hmm ive just follow this https://github.com/apiato/core/blob/master/Abstracts/Requests/Request.php#L259

Reply:
the code you quoted, however, uses whitespace / tabs to enhance readability.. you did not..
i.e., the `explode` is on the same column as `$roles =`.. Thats what i meant..

Reply:
I get it,

Reply:
it is ok? I just refactor it